conceptual/EFCore.PG/mapping/nts.md

```diff
@@ -88,37 +88,52 @@ Note that the plugin is far from covering all spatial operations. If an operatio
 .NET                             | SQL
 ---------------------------------|-----
 geom.Area()                      | [ST_Area(geom)](https://postgis.net/docs/manual-3.0/ST_Area.html)
+geom.AsBinary()                  | [ST_AsBinary(geom)](https://postgis.net/docs/manual-3.0/ST_AsBinary.html)
 geom.AsText()                    | [ST_AsText(geom)](https://postgis.net/docs/manual-3.0/ST_AsText.html)
 geom.Boundary                    | [ST_Boundary(geom)](https://postgis.net/docs/manual-3.0/ST_Boundary.html)
+geom1.Centroid(geom2))           | [ST_Centroid(geom1, geom2)](https://postgis.net/docs/manual-3.0/ST_Centroid.html)
```

Comment:
I think Centroid is a property, not a method accepting another geometry. ]On the PostGIS side](https://postgis.net/docs/ST_Centroid.html) this is just a function accepting a single geomtry.

Reply:
Yes you're right. Fixed in the latest commit.